tests/microos/journal_check.pm

```diff
@@ -20,6 +20,8 @@ use version_utils 'is_opensuse';
 sub run {
     my $self = shift;
 
+    select_console 'root-console';
```

Comment:
The [script_output](https://github.com/os-autoinst/os-autoinst/blob/39d54910a0a664f7302ed30fa88d358813138345/distribution.pm#L235) is sending that ctrl+d because it expects a normal tty, however... since there's nothing within this code that really requires to look at the screen, I'd suggest to use the serial terminal instead... 

```suggestion
    $self->select_serial_terminal;
```

Reply:
Ok, done.